server/common/api/models.py

```diff
@@ -12,6 +12,7 @@ class Class(models.Model):
 	minGuestCount = models.IntegerField(default = 4)
 	maxGuestCount = models.IntegerField(default = 8)
 	price = models.FloatField()
+	classRating = models.FloatField(default = 0.0)
```

Comment:
In order to compute class rating, we need the number of class ratings.
If someone adds rating 3.0 on the class, how can we compute the effect of this rating?

Reply:
add ratingCount column in class table